stable/scalyr/templates/daemonset.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.scalyr.apiKey }}
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
```

Comment:
`{{ template "name" . }}` would be a better choice for the `app` label. I'd suggest you change it everywhere.